mj_envs/hand_manipulation_suite/assets/DAPG_pen.xml

```diff
@@ -65,7 +65,7 @@
             <joint name="OBJRx" pos="0 0 0" axis="1 0 0" limited="false" damping="0" />
             <joint name="OBJRy" pos="0 0 0" axis="0 1 0" limited="false" damping="0" />
             <joint name="OBJRz" pos="0 0 0" axis="0 0 1" limited="false" damping="0" />
-            <geom name="pen" type="cylinder" size="0.015 0.065" condim="4" rgba=".6 .6 .6 .6" />
+            <geom name="pen" type="cylinder" size="0.015 0.065" condim="4" rgba=".6 .6 .6 .6" density="1500" />
```

Comment:
Changing mass can change the final policy's performance. Have we checked if this is a no-op wrt performance?

Reply:
Yes, I have tested it. The committed policies in hand_dapg are for this mass setting. Heavier pen gives smoother and better movements.